plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/file/TestingFileHiveMetastore.java

```diff
@@ -30,6 +30,11 @@
     private TestingFileHiveMetastore() {}
 
     public static FileHiveMetastore createTestingFileHiveMetastore(File catalogDirectory)
+    {
+        return createTestingFileHiveMetastore(catalogDirectory, false);
+    }
+
+    public static FileHiveMetastore createTestingFileHiveMetastore(File catalogDirectory, boolean disableLocationChecks)
```

Comment:
elevating `disableLocationChecks` to `createTestingFileHiveMetastore` overload looks very arbitrary.

instead of this, let's maybe inline this in the caller.
for HdfsEnvironment you may use `io.trino.plugin.hive.HiveTestUtils.HDFS_ENVIRONMENT`, so this won't look very verbose.